types/address/hash.go

```diff
@@ -65,10 +65,9 @@ func Module(moduleName string, key []byte) []byte {
 }
 
 // unsafeStrToByteArray uses unsafe to convert string into byte array. Returned array
-// cannot be altered after this functions is called
+// MUST NOT be altered after this functions is called.
```

Comment:
```suggestion
// must now be altered after this function is called.
```

I'd also add what are the consequences if the caller modifies the return value?

Reply:
the suggestion is not correct. We can't alter the bytes.

Reply:
Obviously, there was a typo. Don't use all caps in go doc comments, please.